flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/JobResultStoreOptions.java

```diff
@@ -65,4 +65,11 @@ public class JobResultStoreOptions {
                                     + "are, instead, marked as clean to indicate their state. In this "
                                     + "case, Flink no longer has ownership and the resources need to "
                                     + "be cleaned up by the user.");
+
+    @Documentation.Section(Documentation.Sections.COMMON_HIGH_AVAILABILITY_JOB_RESULT_STORE)
+    public static final ConfigOption<Integer> TIME_TO_REMOVE_CLEAN_JOB_RESULT =
```

Comment:
I'm wondering whether we want to start a dev ML discussion on that one considering that configuration parameters are considered public API (I guess, a dedicated FLIP would be overkill here). WDYT?

Reply:
I completely agree with your point. We should definitely consider extending this functionality to the file-based JobResultStore. As you suggested, starting a ML discussion seems appropriate. If we decide to expand the scope to the file-based system, following your earlier suggestion, it might be worth considering a FLIP.